task/testmap.py

```diff
@@ -107,6 +107,7 @@
         ],
         # These can be triggered manually with bots/tests-trigger
         '_manual': [
+            'fedora-30/ci',
```

Comment:
"ci" is a rather sad scenario name -- each test is CI, no? What's that supposed to cover? Or is that meant to be a generic placeholder for "random stuff" you want to try in test/run?

Reply:
Yes that's true.. I will find a better name for it when I am done with the test. So yes it's a placeholder for stuff I want to try now manully. It's for this https://github.com/weldr/lorax/pull/882 I want to run pylint and some coverage with it.

Reply:
Note that for such a temporary case you don't need to invent a place-holder -- you can just mutate any existing scenario in test/run to do something different. However, there's nothing specifically bad at this, as long as it doesn't move into the "automatic" section.